db/migrate/20191013121912_add_user_ref_to_tests.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserRefToTests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :tests, :author_id, :integer
+    add_foreign_key :tests, :users, column: :author_id
+    #add_reference(:tests, :author, foreign_key: {to_table: :users, column: :author_id})
```

Comment:
```suggestion
    add_reference(:tests, :author, foreign_key: { to_table: :users })
```
имя колонки вычисляется из второго аргумента, прибавлением постфикса `_id`
а почему закомментировано? вариант верный, его и надо использовать вместо строк 3, 4